images/virtualization-artifact/pkg/controller/cvi/internal/datasource_ready.go

```diff
@@ -89,7 +89,7 @@ func (h DatasourceReadyHandler) Handle(ctx context.Context, cvi *virtv2.ClusterV
 		condition.Reason = cvicondition.VirtualDiskNotReady
 		condition.Message = service.CapitalizeFirstLetter(err.Error())
 		return reconcile.Result{}, nil
-	case errors.As(err, &source.VirtualDiskAttachedToRunningVMError{}):
+	case errors.As(err, &source.VirtualDiskEmployedError{}):
 		condition.Status = metav1.ConditionFalse
 		condition.Reason = cvicondition.VirtualDiskNotReady
```

Comment:
Actually it is Ready, but Employed.

Reply:
In the context of creating an image from a disk to which data can be written, that disk is not ready.

Reply:
I understand, fixed